thorclient/http/client.go

```diff
@@ -0,0 +1,213 @@
+// Copyright (c) 2024 The VeChainThor developers
+
+// Distributed under the GNU Lesser General Public License v3.0 software license, see the accompanying
+// file LICENSE or <https://www.gnu.org/licenses/lgpl-3.0.html>
+
+package http
```

Comment:
http is a pretty commonly used package name, `client` or something else would be better IMO

Reply:
Or maybe just move it inline with `thorclient`?